datasist/datasist/structdata.py

```diff
@@ -75,9 +75,16 @@ def describe(data=None, name='', date_cols=None, show_categories=False, plot_mis
     display(cat_features)
     _space()
     
+    print("\n--------Null Value------")

```

Comment:
The code block display_missing already handles the display of missing values. so this is redundant